### ISSUE TITLE
Harmonize metadata definitions

### ISSUE THREAD

**Issue by stoewer** [zalando-incubator/pazuzu#70]
Currently the feature metadata definitions provided by the [specification of the meta.yml files](https://github.com/zalando-incubator/pazuzu/issues/48) and [storageconnector](https://github.com/zalando-incubator/pazuzu/blob/987f5aae844e1ae9a980c0f169550defb94944ea/storageconnector/storageconnector.go) differ in the data they contain:

meta.yml:
```yaml
description: Java and Node
dependencies:
  - java
  - node
```
FeatureMeta
```go
type FeatureMeta struct {
	Name          string
	Author        string
	CreatedAt     time.Time
	UpdatedAt     time.Time
	Dependencies []string
}
```

Therefore the following changes should be made:
* Add a field `Description` to `FeatureMeta`
* Add the field `author` to `meta.yml`
* Remove `CreatedAt` from `FeatureMeta` since this information is not available via go-git

**Comment by nayef** [comment]
should author field be optional in the yml?

**Comment by stoewer** [comment]
So far technically all fields are optional, but it could be a good idea to make some fields mandatory.

**Comment by stoewer** [comment]
Was resolved by #70
